CMakeLists.txt

```diff
@@ -143,29 +143,13 @@ include_optional_module(ENVIRONMENT_VARIABLE WITH_VARIABLE_SPLASH_SCREEN
 include_optional_module(ENVIRONMENT_VARIABLE WITH_OWN_QTKEYCHAIN
                         OPTION_VARIABLE USE_OWN_QTKEYCHAIN
                         READABLE_NAME "own QtKeychain library")
-include_optional_module(ENVIRONMENT_VARIABLE WITH_QT6
```

Comment:
Since we've cut out the Qt 5 stuff it is no longer optional to use Qt 6 - of course this will clash with any existing PR to remove Qt5 but I couldn't see a PR for that open.